src/app.ts

```diff
@@ -16,9 +16,6 @@ async function main() {
   const env = Object.assign({}, process.env);
   const command = argv._[0];
 
-  // See https://github.com/grafana/grafana-image-renderer/issues/460
```

Comment:
This is no longer needed. We are using the `shell` value instead of the new headless mode.

Reply:
Good catch! This is also set in the tests: https://github.com/grafana/grafana-image-renderer/blob/master/src/service/http-server.integration.test.ts#L101